uint32  id                # id сообщения 
Struct_Lidar[4] lidar     # Структура дальномеров